### PR TITLE
[HTML5] - Update client styles

### PR DESCRIPTION
**Pull request by KDSBrowne**
This will update the element styles for the message input, list items and the public chat options button.
   
message-input

list-items

more

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/styles.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import "/imports/ui/components/user-list/styles.scss";
 @import "/imports/ui/stylesheets/mixins/_scrollable";
+@import "../../../stylesheets/variables/focus";
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/styles.scss]: Please import via absolute location `/imports/ui/xxxxxxxxxxxxxx`

File: bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss [hunk 2]
```diff
@@ -13,3 +13,7 @@ $lg-padding-y: .5rem;
 
 $jumbo-padding-x: 3.025rem;
 $jumbo-padding-y: 1.25rem;
+
+$icon-top-offset: -.4em;
+$icon-left-offset: -.1em;
+$square-side-length: 1.56rem;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss]: Why those variables are in the general section?

Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss]: Wasn't sure if I was going to need to use them again for other elements. I'll move them back to the `chat-dropdown/styles.scss` where they are currently used.

**Comment by antobinary** [comment]
Closing PR as it contains code from BBB 2.1 (master) but targets BBB 2.0

**Pull request closed without merging**